RecoLocalTracker/SiStripClusterizer/plugins/SiStripRawToClusterGPUKernel.cu

```diff
@@ -0,0 +1,551 @@
+#include <cub/cub.cuh>
+
+#include "HeterogeneousCore/CUDAUtilities/interface/allocate_device.h"
+#include "HeterogeneousCore/CUDAUtilities/interface/allocate_host.h"
+#include "HeterogeneousCore/CUDAUtilities/interface/currentDevice.h"
+
+#include "HeterogeneousCore/CUDAUtilities/interface/host_unique_ptr.h"
+#include "HeterogeneousCore/CUDAUtilities/interface/device_unique_ptr.h"
+
+#include "HeterogeneousCore/CUDAUtilities/interface/copyAsync.h"
+
+#include "CUDADataFormats/SiStripCluster/interface/SiStripClustersCUDA.h"
+
+#include "CalibFormats/SiStripObjects/interface/SiStripClusterizerConditionsGPU.h"
+
+#include "ChannelLocsGPU.h"
+#include "SiStripRawToClusterGPUKernel.h"
+#include "StripDataView.cuh"
+
+//#define GPU_DEBUG
+#if defined(EDM_ML_DEBUG) || defined(GPU_DEBUG)
+#define GPU_CHECK
```

Comment:
is this intended to be kept in?

Reply:
GPU_CHECK is used in several spots. Did you prefer repeating these two conditions instead?